src/test/apiTest/java/uk/gov/hmcts/ethos/replacement/apitest/config/TestCacheConfig.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.ethos.replacement.apitest.config;
+
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.concurrent.ConcurrentMapCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Primary;
+
+@TestConfiguration
+public class TestCacheConfig {
```

Comment:
We don't need this configuration class. @CacheConfig(cacheNames = {"adminUserToken"}) on AdminUserService exactly does what is required to configure caching.

Reply:
This is added to fix the tests as its require full app context

Reply:
You don't need to harpreet, You can implement tests as in [cachedIdamServiceTest](https://github.com/hmcts/et-sya-api/pull/1324/files#diff-19df789f4f2cd6cfb4f217219867594087d0ebab7b089fea09977c3643d88b5a). Make all of them in one test class. 

But it is up to you. Everyone’s got their own style.